protobuf/crypto/dkg/dkg.proto

```diff
@@ -9,13 +9,16 @@ package dkg;
 option go_package = "github.com/drand/drand/protobuf/crypto/dkg";
 /*option go_package = "dkg";*/
 
+import "common/common.proto";
+
 // Packet is a wrapper around the three different types of DKG messages
 message Packet {
     oneof Bundle {
         DealBundle deal = 1;
         ResponseBundle response = 2;
         JustificationBundle justification = 3;
     }
+    common.Context context = 4;
```

Comment:
If we keep this we should put it at the protocol.proto level, it's nothing crypto related.